code/modules/emotes/definitions/human.dm

```diff
@@ -0,0 +1,62 @@
+/decl/emote/human
+	key = "vomit"
+
+/decl/emote/human/check_user(var/mob/living/carbon/human/user)
+	return (istype(user) && user.check_has_mouth() && !user.isSynthetic())
+
+/decl/emote/human/do_emote(var/mob/living/carbon/human/user)
+	user.vomit()
```

Comment:
No, the proc already handles a cooldown. I am not adding a mechanic, you can already do `*vomit`.

Reply:
Why is this one on the **/decl/emote/human _parent type_**???

Reply:
Looks like an oversight in the original implementation. Hasn't been an issue because all of the subtypes override that proc but should probably be fixed.